assets/blocks/BlueberryBushFull.block

```diff
@@ -1,8 +0,0 @@
-+{
```

Comment:
Problem with your branch here - since you started by adding blueberries (clashing with existing content) then deleted / renamed your stuff into blackberries this file got deleted in the crossfire, which we don't want, since then the original blueberry will break.

I see a merge commit in your branch, that's generally a bad sign and a hint that using a fresh new branch with content copy pasted in would be advisable.